### PR TITLE
[chore](UT) Add UT for cloud string util

### PR DESCRIPTION
**Pull request by gavinchou**

### CODE REVIEW

**Comment by gavinchou** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

**Comment by gavinchou** [comment]
run buildall

**Comment by gavinchou** [comment]
run buildall

**Comment by gavinchou** [comment]
run buildall

**Comment by gavinchou** [comment]
run cloud_ut

**Comment by gavinchou** [comment]
run buildall

**Comment by gavinchou** [comment]
run buildall

**Comment by gavinchou** [comment]
run cloud_ut

**Comment by dataroaring** [comment]
run buildall

**Comment by gavinchou** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

File: cloud/test/util_test.cpp [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+        ++idx;
+    }
+
+    idx = 0;
+    std::vector<std::tuple<std::string, std::string>> trim_inputs {
+        {""              , ""     },
+        {""              , ""     },
+        {"/"             , ""     },
+        {"\t////"        , ""     },
+        {"/a ///"        , "a "   },
```
Comment (gavinchou) [inline cloud/test/util_test.cpp:76]:
```suggestion
        {"/a ///"        , "a"   },
```

**Comment by gavinchou** [comment]
run buildall

**Review by ByteYue — APPROVED** [review]

**Comment by gavinchou** [comment]
run buildall

**Review by bobhan1 — APPROVED** [review]

**Review by Yukang-Lian — APPROVED** [review]

**Review by ByteYue — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**